cognite/neat/_session/_prepare.py

```diff
@@ -50,6 +60,47 @@ def __init__(self, state: SessionState, verbose: bool) -> None:
         self._state = state
         self._verbose = verbose
 
+    def dexpi(self) -> None:
+        """Prepares extracted DEXPI graph for further usage in CDF
+
+        This method bundles several graph transformers which:
+        - attach values of generic attributes to nodes
+        - create associations between nodes
+        - remove unused generic attributes
+        - remove associations between nodes that do not exist in the extracted graph
+        - remove edges to nodes that do not exist in the extracted graph
+
+        and therefore safeguard CDF from a bad graph
+        """
+
+        DEXPI = get_default_prefixes_and_namespaces()["dexpi"]
```

Comment:
Not a global constant
```suggestion
        dexpi = get_default_prefixes_and_namespaces()["dexpi"]
```